packages/common/container-definitions/src/loader.ts

```diff
@@ -196,14 +308,17 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
     getLoadedCodeDetails(): IFluidCodeDetails | undefined;
 
     /**
-     * Returns true if the container has been closed, otherwise false
+     * Returns true if the container has been closed, otherwise false.
      */
     readonly closed: boolean;
 
     /**
-     * Returns true if the container is dirty, i.e. there are user changes that has not been saved
-     * Closing container in this state results in data loss for user.
-     * Container usually gets into this situation due to loss of connectivity.
+     * Whether or not there there are any local changes that have not been saved.
```

Comment:
double "there"
I think it's important to clarify RE "saved where?". Currently, it's about storage only.
Which means your remarks are not accurate, as user scenario may include user serializing container state locally and rehydrating it later, thus preserving such changes across sessions (and eventually pushing to storage)

Reply:
```suggestion
     * Whether or not there are any local changes that have not been saved.
```